src/customizations/components/theme/AppExtras/AppExtras.jsx

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
+import { Helmet } from '@plone/volto/helpers';
 import ScrollToTop from '@italia/components/ItaliaTheme/ScrollToTop/ScrollToTop';
 import { SubsiteLoader } from '@italia/addons/volto-subsites';
+import { siteConfig } from '~/config';
 
 const AppExtras = ({ pathname }) => {
+  const subsite = useSelector((state) => state.subsite?.data);
+  const content = useSelector((state) => state.content?.data);
+
   return (
     <>
+      <Helmet>
+        <title>
+          {subsite?.title || siteConfig.properties.siteTitle} - {content.title}
```

Comment:
```suggestion
          {subsite?.title ?? siteConfig.properties.siteTitle} - {content.title}
```
Non è più corretto così? 🤔